src/test/java/org/entando/plugins/pda/core/service/process/FakeProcessService.java

```diff
@@ -25,7 +25,7 @@
     public static final String PROCESS_NAME_1 = "Process 1";
     public static final String PROCESS_PROP_1 = "New Prop 1";
     public static final String PROCESS_DIAGRAM_FILENAME_1 = "process_diagram_1.svg";
-    public static final int PROCESS_DIAGRAM_LENGTH_1 = 63_453;
+    public static final int PROCESS_DIAGRAM_LENGTH_1 = 63_945;
```

Comment:
Test needs to be fixed.
Is this file returning differente length on windows?
The correct should be `63_ 453`

Reply:
The curren test version passes on Jenkins, but not in my machine. Does the tests pass on yours? If so, you might be right and doesn't work on Windows

Reply:
Yes, they work on my machine.